cmd/gardener-extension-acl/app/app.go

```diff
@@ -80,15 +81,17 @@ func (o *Options) run(ctx context.Context) error {
 		return fmt.Errorf("could not update manager scheme: %s", err)
 	}
 
-	if err := istionetworkingClientGo.AddToScheme(mgr.GetScheme()); err != nil {
+	if err := istionetworkv1alpha3.AddToScheme(mgr.GetScheme()); err != nil {
+		return fmt.Errorf("could not update manager scheme: %s", err)
+	}
+	if err := istionetworkv1beta1.AddToScheme(mgr.GetScheme()); err != nil {
```

Comment:
This is used for the `Gateway` object. Any reason why for the `EnvoyFilter` object, we use `istionetworkv1alpha3`?

Reply:
Because that's the respective versions that are used in the Gardener Seed cluster :smile: